transactions/privateForwarder/deploy_forwarder_contract.cdc

```diff
@@ -0,0 +1,9 @@
+transaction(contractName: String, code: [UInt8], senderStoragePath: StoragePath, storagePath: StoragePath, publicPath: PublicPath) {
+
+  prepare(signer: AuthAccount) {
+
+    signer.contracts.add(name: contractName, code: code, senderStoragePath, storagePath, publicPath)
```

Comment:
This looks like a standard contract deploy TX. How come it is in privateForwarder?

Reply:
because we use it to deploy the forwarder contract and it uses arguments for the forwarder contract. Is there a more generic way to pass arguments to the generic create account functionality in the Go SDK?

Reply:
Ok, I see! Missed that part. Yeah I guess there is no template like that yet.